src/main/java/com/greenfoxacademy/goddesstribesbackend/services/BuildingService.java

```diff
@@ -41,4 +49,32 @@ public Mine saveMineAtStart(Kingdom kingdom) {
     return null;
   }
 
+  public ArrayList<Farm> findFarms(Long kingdomId) {
+    return farmRepository.findFarmsByKingdom_Id(kingdomId);
+  }
+
+  public ArrayList<Mine> findMines(Long kingdomId) {
+    return mineRepository.findMinesByKingdom_Id(kingdomId);
+  }
+
+  public int findFoodProductionRate(Long kingdomId) {
```

Comment:
nem `generation`-nek hivjuk? es nem keressuk (tehat nem find) hanem `getFoodGeneration`

Reply:
Kedves Barni! Igazad van, generation-nek hívjuk a dto -ban, de nekem ez nem tetszett... a kódban, amit nem lát a frontend productionRate-nek hívtam. Legyen a kettő egyforma?
A getFoodGeneration-ért major issue-t kaptam Krisztiántól a január 18-i alapvizsgán (és 1 pont levonást), mert szerinte nem kezdünk get-tel metódusnevet, ha abban van "logika"...(Nekem is jobban tetszene get-tel). Most mi legyen??